lib/core/Canvas.js

```diff
@@ -515,14 +515,19 @@ Canvas.prototype.toggleMarker = function(element, marker) {
 };
 
 /**
- * Returns the current root element.
```

Comment:
Could we rename the commit to more clearly indicate what it does?

> fix(canvas): do not implicitly create root if roots already exist

Could also add a description on how we have two distinct modes (nothing inbetween) 

* magic, implicit where an editor does not care about roots
* controlled where the toolkit building on top does care about roots and manages them explicitly (no magic at all)